rs/sns/governance/src/proposal.rs

```diff
@@ -442,6 +466,10 @@ pub(crate) async fn validate_and_render_action(
         proposal::Action::ManageDappCanisterSettings(manage_dapp_canister_settings) => {
             validate_and_render_manage_dapp_canister_settings(manage_dapp_canister_settings)
         }
+        proposal::Action::AdvanceSnsTargetVersion(_) => {
+            // TODO[NNS1-3434]: Implement Action::AdvanceSnsTargetVersion
+            return Err("Action::AdvanceSnsTargetVersion is not implemented yet.".to_string());
```

Comment:
If you want to implement this before the feature is ready for release, you can consider using `self.check_test_features_enabled();` which will panic if we are not in a test configuration

Reply:
Thanks for the pointer, that might be helpful for the next PR.